assets/panel/debugger.properties

```diff
@@ -332,6 +332,15 @@ searchPanelGoToLine=Go to line (%S)
 # filter panel popup for the variables search operation.
 searchPanelVariable=Filter variables (%S)
 
+# LOCALIZATION NOTE (breakpointHeadingMenuItem): The text for all the elements
```

Comment:
This comment will be displayed only for the first string, but I don't think there's a lot of value in adding it to all strings. For clarity, maybe `# LOCALIZATION NOTE (breakpointHeadingMenuItem.*)`